techlibs/gatemate/synth_gatemate.cc

```diff
@@ -71,6 +71,9 @@ struct SynthGateMatePass : public ScriptPass
 		log("    -dff\n");
 		log("        run 'abc' with -dff option\n");
 		log("\n");
+		log("    -initial\n");

```

Comment:
I think there could be a better name for this like `-ignore-initial` or similar?

Reply:
Yep. I'm usually really bad to find a good option name :-/

Maybe `-ignore-initial-value`: to be more explicit?
If it's good to you I update my PR.
Thanks

Reply:
fixed